### PR TITLE
make a curriculum vitae

### PR DESCRIPTION
**Pull request by viabadia**

### CODE REVIEW

**Comment by Ury18** [comment]
Ayy lmao

File: staff/victor-abadia/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    <th>Dates</th>
+                    <td>2016-2019</td>
+                </tr>
+            </table>
+        </section>
+    </section>
+    <br>
+    <footer>
+        <h3>Contact</h3>
+        <p>e-mail: Eddard.Stark@7kingdoms.com</p>
```
Comment (msarroca) [inline staff/victor-abadia/cv/index.html]: If you want, you can use: <a><a href="mailto:rick@pawn.com">Email</a></a> to open directly the mail client

File: staff/victor-abadia/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    <header>
+        <h1>Eddard Stark</h1>
+    </header>
+    <br>
+    <nav>
+        <a href="#personal"><h4>Personal Data</h4></a>
+        <a href="#education"><h4>Education</h4></a>
+        <a href="#employment"><h4>Employment</h4></a>
+    </nav>
+    <br>
```
Comment (robert-z) [inline staff/victor-abadia/cv/index.html]: Unnecessary `<br>` tag

File: staff/victor-abadia/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    <section>
```
Comment (robert-z) [inline staff/victor-abadia/cv/index.html]: I would change this `<section>` tag for a `<main>` tag

Comment (johanaludwig) [reply staff/victor-abadia/cv/index.html]: here the link: https://developer.mozilla.org/es/docs/Web/HTML/Elemento/main

File: staff/victor-abadia/cv/index.html (hunk 2 above)
Comment (gcelma) [reply staff/victor-abadia/cv/index.html]: I would add the nav inside the header!

File: staff/victor-abadia/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                    <td>The Throne</td>
+                    <th>Title</th>
+                    <td>Hand of the King</td>
+                    <th>Dates</th>
+                    <td>2016-2019</td>
+                </tr>
+            </table>
+        </section>
+    </section>
+    <br>
```
Comment (martimalek) [inline staff/victor-abadia/cv/index.html]: `<br>` tag shouldn't be used!

**Pull request merged**